R/dm-from-src.R

```diff
@@ -20,10 +20,12 @@
 #'   `r lifecycle::badge("experimental")`
 #'
 #'   Additional parameters for the schema learning query.
-#'   Currently supports `schema` (default: `"public"`)
-#'   and `table_type` (default: `"BASE TABLE"`) for Postgres databases.
 #'
-#'   For MSSQL databases `schema` (default: `"dbo"`) is supported.
+#'   - `schema`: supported for MSSQL (default: `"dbo"`) and Postgres (default: `"public"`).
+#'   Learn the tables in a specific schema.
+#'   - `dbname`: supported for MSSQL. Access different databases on the connected MSSQL-server;
+#'   default: database addressed by `src`.
+#'   - `table_type`: supported for Postgres (default: `"BASE TABLE"`). Specify the table type.
```

Comment:
What are other possible options here?

Reply:
we are currently only supporting those 3. Anything else in the ellipsis will be ignored.

Reply:
What options other than `"BASE TABLE"` exist?